OpenIPSL/Interfaces.mo

```diff
@@ -2,7 +2,7 @@ within OpenIPSL;
 package Interfaces
```

Comment:
Now I'm surprised moving the PwPin didn't cause problems in the Application examples... This should have shown up in the CI if it were the case.